ccd-definition/CaseField/CareSupervision/caseDocuments.json

```diff
@@ -0,0 +1,96 @@
+[
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "ID": "hearingFurtherEvidenceDocuments",
+    "FieldType": "Collection",
+    "FieldTypeParameter": "HearingFurtherEvidenceBundle",
+    "Label": "Further evidence documents for hearings",
+    "SecurityClassification": "Public"
```

Comment:
could add some comments to these fields for more clarity, as it is quite confusing for anyone not familiar